src/completion.py

```diff
@@ -53,11 +54,11 @@ async def generate_completion_response(
         )
         rendered = prompt.full_render(MY_BOT_NAME)
         response = await client.chat.completions.create(
-            model=MODEL,
+            model=thread_config.model,
             messages=rendered,
-            temperature=1.0,
+            temperature=thread_config.temperature,
             top_p=0.9,
```

Comment:
Sometimes when temperature varies and top_p isn't 1.0 the output gets weird, should we fix top_p to 1? I chose temperature to be a param here and omitted having top_p as one just because temperature was the one most people were familiar with and we didn't want people messing with both at the same time